diff --git a/node_modules/react-native-phone-number-input/lib/index.js b/node_modules/react-native-phone-number-input/lib/index.js
index b89c388..b4edf77 100644
--- a/node_modules/react-native-phone-number-input/lib/index.js
+++ b/node_modules/react-native-phone-number-input/lib/index.js
@@ -180,6 +180,8 @@ export default class PhoneInput extends PureComponent {
             disabled={disabled}
             onPress={() => this.setState({ modalVisible: true })}
           >
+            {disabled === true ? 
+            <></>: 
             <CountryPicker
               onSelect={this.onSelect}
               withEmoji
@@ -188,13 +190,13 @@ export default class PhoneInput extends PureComponent {
               filterProps={filterProps}
               countryCode={countryCode}
               withCallingCode
-              disableNativeModal={disabled}
+              disableNativeModal={false}
               visible={modalVisible}
               theme={withDarkTheme ? DARK_THEME : DEFAULT_THEME}
               renderFlagButton={this.renderFlagButton}
               onClose={() => this.setState({ modalVisible: false })}
               {...countryPickerProps}
-            />
+            />}
             {code && layout === "second" && (
               <Text
                 style={[styles.codeText, codeTextStyle ? codeTextStyle : {}]}
